support/oidc-discovery-provider/main.go

```diff
@@ -41,11 +42,22 @@ func run(configPath string) error {
 	var source JWKSSource
 	switch {
 	case config.RegistrationAPI != nil:
-		source, err = NewRegistrationAPISource(RegistrationAPISourceConfig{
+		log.Warn("The registration_api configuration is deprecated in favor of server_api; please update your configuration")
```

Comment:
I'm wondering if we should be explicit about what "deprecated" means. I think that we usually say that "it will be removed in a future release". Just to provide an indication of what is the consequence of this.

Something like:
```suggestion
		log.Warn("The registration_api configuration is deprecated in favor of server_api and will be removed in a future release; please update your configuration")
```

Reply:
:+1: